README.md

```diff
@@ -1,8 +1,163 @@
 # Substrait
```

Comment:
If we intend to call this package PySubstrait, I think it's best for the heading at the top of the readme to use that name
```suggestion
# PySubstrait
```